actions/sequester/ImportIssues/Program.cs

```diff
@@ -26,6 +22,9 @@ private static async Task<int> Main(string org, string repo, int? issue = null,
                 repo = split[1];
             }
 
+            Console.WriteLine(issue.HasValue && issue > -1
+                ? $"Processing single issue {issue.Value}: https://github.com/{org}/{repo}/issues/{issue.Value}"
+                : "Processing all open issues");
```

Comment:
Maybe add org and Repo?

```suggestion
                : $"Processing all open issues: {org}/{repo}");
```